longevity_operator_multi_tenant_test.py

```diff
@@ -0,0 +1,101 @@
+        self.db_cluster = db_cluster
+        self.loaders = loaders
+        self.monitors = monitors
+        self.prometheus_db = prometheus_db
+        self.params = copy.deepcopy(params)
+        self.test_config = test_config
+        self._duration = self.params.get(key='test_duration')
+        self.log = logging.getLogger(self.__class__.__name__)
+        self.create_stats = self.params.get(key='store_perf_results')
+        self.status = "RUNNING"
+        self.cluster_index = str(cluster_index)
+        self._test_id = self.test_config.test_id() + f"--{cluster_index}"
+        self._test_index = self.get_str_index()
+        self.start_time = self.get_test_start_time()
+        self._duration = self.params.get(key='test_duration')
+        self.start_time = time.time()
+        self.timeout_thread = self._init_test_timeout_thread()
+        self.test_config.reuse_cluster(False)
+
+    def get_str_index(self):
+        return f"k8s-longevity-{self.db_cluster.k8s_cluster.tenants_number}-tenants"
+
+    def id(self):  # pylint: disable=invalid-name
+        return self.test_config.test_id() + f"-{self._test_index}"
+
+    def __str__(self) -> str:
+        return self._test_index + f"--{self.cluster_index}"
+
+    def __repr__(self) -> str:
+        return self.__str__()
+
+
+class LongevityOperatorMulitiTenantTest(LongevityTest):
+    scylla_clusters_stats = []
+    _testMethodName = "runTest"
+
+    def setUp(self):
+        super().setUp()
+        for i, db_cluster in enumerate(self.db_clusters_multitenant):  # pylint: disable=no-member
+            self.scylla_clusters_stats.append(ScyllaClusterStats(
+                db_cluster=db_cluster,
+                loaders=self.loaders_multitenant[i],  # pylint: disable=no-member
+                monitors=self.monitors_multitenant[i],  # pylint: disable=no-member
+                prometheus_db=self.prometheus_db_multitenant[i],  # pylint: disable=no-member
+                params=self.params,
+                test_config=self.test_config,
+                cluster_index=i + 1,
+            ))
+            for stress_cmd_param in self.params.stress_cmd_params:
+                current_stress_cmd = self.params.get(stress_cmd_param)
+                if not isinstance(current_stress_cmd, list):
+                    continue
+                # NOTE: 'prepare_write_cmd' is allowed to be list of strs
+                if stress_cmd_param == 'prepare_write_cmd':
+                    if not all((isinstance(current_stress_cmd_element, list)
+                                for current_stress_cmd_element in current_stress_cmd)):
+                        continue
+                self.scylla_clusters_stats[i].params[stress_cmd_param] = current_stress_cmd[i]
+
+    def test_custom_time(self):
+        def _run_test_on_one_tenant(scylla_cluster_stats):
+            self.log.info("Longevity test with parameters: %s", scylla_cluster_stats.params)
```

Comment:
Would be good to add unique part of the cluster in the log message. For example - id.